### DISCUSSION TITLE
Purchased 2 premiums only 1 working

### DISCUSSION THREAD

**Discussion by uaedude** [tribalfs/GalaxyMaxHzPub#232] [Q&A] [answered]
Hi Dev & all
I have purchased two premiums one for the galaxy z flip 5 and one for the galaxy tab S8 plus, with the tablet it shows premium. However, I cannot alter the hz like the z flip 5 and some features doesn't work. 
The other device zflip5 in the app I don't see a premium subscription, I even thought it must be premium because it gave me a notification many times in the past but still not in the app>settings>about galaxymaxhz
I have even looked for the option to change the coverscreen hz and it's not their that means I'm still in standard edition.

Regards with much love and appreciation;

_> love ur great works dev &i wish you be successful toall projects vastly with serenity in ur lifestyle, Thank you._

**Comment by tribalfs** [comment, marked as answer]
Hi!
Please [email](mailto:tribalfs@gmail.com?subject=[GitHub]%20Galaxy%20MaxHz) us the activation code so we can check. Thank you.
